CMakeLists.txt

```diff
@@ -28,6 +28,12 @@ set(CMAKE_VERBOSE_MAKEFILE OFF)
 
 get_filename_component(REPOSITORY_DIR ${PROJECT_SOURCE_DIR} ABSOLUTE)
 
+# gcc v4.9 requires its own binutils-wrappers for LTO (flag -flto)
+# fixes #981
+IF(UNIX AND CMAKE_COMPILER_IS_GNUCXX AND (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "4.9" OR CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL "4.9"))
+    set(CMAKE_AR "/usr/bin/gcc-ar")
+    set(CMAKE_RANLIB "/usr/bin/gcc-ranlib")
```

Comment:
Will remove the absolute path in CMAKE_AR and CMAKE_RANLIB.
Both variables seem to be used in the config process (if you remove the variables, you'll get a lot of "lto plugin needed" errors and build fails).